Program.cs

```diff
@@ -26,9 +26,9 @@ static void Main(string[] args)
             {
                 var nancyHost = new Nancy.Hosting.Self.NancyHost(new Uri(httpUrl), new CustomBootstrapper());
                 nancyHost.Start();
-                Console.WriteLine("Web server running...");
+                Console.WriteLine("Web server running at " + httpUrl);
 
-                Process.Start(httpUrl);
+                /* Process.Start(httpUrl); */
```

Comment:
Why commented?

Reply:
Why commented and not deleted?
I don't remember.
The line can be deleted.